src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -67,14 +75,25 @@ def set_scp_attachment(
                         path)
                     pass
 
+    @staticmethod
+    def return_policy_name(policy_type, target_path, policy_filename):
+        _type = 'scp' if policy_type == "SERVICE_CONTROL_POLICY" else 'tagging-policy'
+        if policy_filename != '{}.json':
+            return 'adf-{0}-{1}--{2}'.format(_type, target_path, policy_filename)
+        else:
+            # Added for backwards-compatibility with previous versions of ADF
+            return 'adf-{0}-{1}'.format(_type, target_path)
```

Comment:
The file name is not inserted into the string at the moment.
I think it needs to be changed to an if against a format string.

Additionally, it will not continue code execution after a return, so we can shorten it too by removing the `else`.

Considering the longer string in the if statement, I would propose we flip those, so we skip indenting the longest line.

All together, you would have something like:

```suggestion
        if policy_filename == f'{_type}.json':
            # Added for backwards-compatibility with previous versions of ADF
            return 'adf-{0}-{1}'.format(_type, target_path)
        return 'adf-{0}-{1}--{2}'.format(_type, target_path, policy_filename)
```